platform/linuxbsd/display_server_x11.cpp

```diff
@@ -753,6 +754,38 @@ void DisplayServerX11::window_set_title(const String &p_title, WindowID p_window
 	XChangeProperty(x11_display, wd.x11_window, _net_wm_name, utf8_string, 8, PropModeReplace, (unsigned char *)p_title.utf8().get_data(), p_title.utf8().length());
 }
 
+void DisplayServerX11::window_set_mouse_passthrough(const Vector<Vector2> &p_region, WindowID p_window) {
+	_THREAD_SAFE_METHOD_
+
+	ERR_FAIL_COND(!windows.has(p_window));
+	WindowData &wd = windows[p_window];
```

Comment:
This could be const.
```suggestion
	const WindowData &wd = windows[p_window];
```

Reply:
> I'm not sure what real life use cases are for this feature.

Summoning ancient horror.
![clippy](https://user-images.githubusercontent.com/7645683/93451745-3db73100-f8e0-11ea-876a-ccf61f0c7463.png)

Old Widows transparency had click-through as side effect and was used for something, AFAIK no one requested this feature on Linux.